packages/paste-website/src/components/ContactUsMenu.tsx

```diff
@@ -16,33 +18,45 @@ export const ContactUsMenu: React.FC<{}> = () => {
   };
 
   return (
-    <>
-      <MenuButton {...menu} variant="link">
-        Contact us <ChevronDownIcon decorative />
-      </MenuButton>
-      <Menu {...menu} aria-label="Contact us">
-        <MenuItem
-          {...menu}
-          href="https://github.com/twilio-labs/paste/discussions"
-          onClick={() => handleClick('Contact Menu', 'Ask a question')}
-        >
-          Ask a question
-        </MenuItem>
-        <MenuItem
-          {...menu}
-          href="https://github.com/twilio-labs/paste/discussions"
-          onClick={() => handleClick('Contact Menu', 'Request a feature')}
-        >
-          Request a feature
-        </MenuItem>
-        <MenuItem
-          {...menu}
-          href="https://github.com/twilio-labs/paste/issues/new?assignees=&labels=Type%3A+Bug&template=bug_report.md&title="
-          onClick={() => handleClick('Contact Menu', 'Report a bug')}
-        >
-          Report a bug
-        </MenuItem>
-      </Menu>
-    </>
+    <Box position="fixed" bottom={0} left={0}>
+      <Box backgroundColor="colorBackground" padding="space20">
+        <MenuButton {...menu} variant="reset" size="reset">
+          {/* 212px / 13.25rem minWidth because of the boxshadow focus state being cut off by the sidebar width and edge of browser.
+        sizeSidebar10 - 8px =  212px / 13.25rem. */}
+          <Box as="span" display="flex" alignItems="center" padding="space40" minWidth="13.25rem">
```

Comment:
rather than do all this, can't you just put it in a box with 8px padding?

Reply:
Refactored so no longer need the width hack.